test/unit/range/views/view_pairwise_combine_test.cpp

```diff
@@ -353,11 +353,21 @@ TYPED_TEST(pairwise_combine_test, view_concept)
     EXPECT_TRUE(std::ranges::input_range<typename TestFixture::view_t>);
     EXPECT_TRUE(std::ranges::forward_range<typename TestFixture::view_t>);
     EXPECT_TRUE(std::ranges::view<typename TestFixture::view_t>);
-    EXPECT_FALSE((std::ranges::output_range<typename TestFixture::view_t, std::tuple<char &, char &>>));
+    EXPECT_TRUE((std::ranges::output_range<typename TestFixture::view_t, std::tuple<char &, char &>>));
     EXPECT_EQ(std::ranges::bidirectional_range<TypeParam>,
               std::ranges::bidirectional_range<typename TestFixture::view_t>);
     EXPECT_EQ(std::ranges::sized_range<TypeParam>, std::ranges::sized_range<typename TestFixture::view_t>);
     EXPECT_EQ(std::ranges::random_access_range<TypeParam>, std::ranges::random_access_range<typename TestFixture::view_t>);
+
+    EXPECT_TRUE(std::ranges::input_range<typename TestFixture::const_view_t>);
+    EXPECT_TRUE(std::ranges::forward_range<typename TestFixture::const_view_t>);
+    EXPECT_TRUE(std::ranges::view<typename TestFixture::const_view_t>);
+    //TODO: Investigate, because this should be false. It cannot be used as a output range!
```

Comment:
That is the magic of the common_tuple/common_pair 🐰 🎩 . With a regular std::tuple this does not work, because of the writable concept which requires that `const proxy_t && = value_t` works.
> writable has the awkward const_­cast expressions to reject iterators with prvalue non-proxy reference types that permit rvalue assignment but do not also permit const rvalue assignment.
Consequently, an iterator type I that returns std​::​string by value does not model writable<I, std​::​string>.


And you want it to be an output range like e.g. the zip view, so you can do something like:
```cpp
auto zipped = views::zip(rng1, rng2, ...);
for (auto && elem : zipped)
   elem = std::tuple{val1, val2, ...};
```
So the zip view should preserve output_range if all of the contained ranges are output_ranges. To solve the todo, I would first wait for the ranges update and then it is probably a little nasty to find the reasons for this. I was looking into implementing the common_pair/tuple for our lib (we need anyway). And see if that helps. But that is beyond of this PR.